modules/twinkleprod.js

```diff
@@ -445,14 +428,19 @@ Twinkle.prod.callback.evaluate = function twinkleprodCallbackEvaluate(e) {
 	Morebits.status.init( form );
 
 	var talk_title = new mw.Title(mw.config.get('wgPageName')).getTalkPage().getPrefixedText();
+	// Talk page templates for PROD-able discussions
+	var blocking_templates = 'Template:Old XfD multi|Template:Old MfD|Template:Oldffdfull|' + // Common prior XfD talk page templates
```

Comment:
```suggestion
	var blocking_templates = 'Template:Old XfD multi|Template:Old AfD|Template:Old MfD|Template:Oldffdfull|' + // Common prior XfD talk page templates
```
lol, it looks like you forgot the most important one.

Reply:
That's the neat part — {{Old AfD}} just transcludes a single instance of {{Old AfD multi}}, which is a redirect to {{Old XfD multi}}, so there's no need to explicitly include it.  Check out [this query](https://en.wikipedia.org/wiki/Special:ApiSandbox#action=query&format=json&prop=templates&titles=Talk%3AOrder%2066%20(Star%20Wars)&tltemplates=Template%3AOld%20XfD%20multi%7CTemplate%3AOld%20MfD).

Reply:
Oh ok, didn't notice {{old AfD}} was implemented like that, though the other {{old [x]fd}} templates aren't.